src/apis/api.js

```diff
@@ -0,0 +1,27 @@
+// https://bootcamp-api.codeit.kr/api/sample/user
+
+const baseUrl = 'https://bootcamp-api.codeit.kr/api';
+
+export const getProfileData = async () => {
+  const res = await fetch(`${baseUrl}/sample/user`).then((res) => res.json());
+
+  return res;
```

Comment:
P3:
변수에 할당할 이유가 없어보이네요~

```suggestion
   return await fetch(`${baseUrl}/sample/user`).then((res) => res.json());
```